docs/cli-commands.md

```diff
@@ -9,18 +9,39 @@ Most of the commands documented on this page
 assume that you know how to address the subdomains of a DSP server.
 There are three relevant URLs you should know about:
 
-- Subdomain `admin` stands for the DSP-APP frontend that you look at in your browser
+- Subdomains `admin`/`app` stand for the DSP-APP frontend that you look at in your browser
 - Subdomain `api` stands for the DSP-API (where DSP-TOOLS sends its data to) 
 - Subdomain `iiif` stands for the SIPI server interface (where DSP-TOOLS sends the multimedia files to)
 
-This means that for uploading data to a DSP server 
+This means that for uploading data to the DSP server 
 on the domain `dasch.swiss`, 
 you have to type the following:
 
 ```bash
-dsp-tools xmlupload -s https://api.dasch.swiss -u root@example.com -p 'test' -S https://iiif.dasch.swiss xml_data_file.xml
+dsp-tools xmlupload -s https://api.dasch.swiss -u 'your@email.com' -p 'password' xml_data_file.xml
 ```
 
+If the user input is not correct,
+DSP-TOOLS tries to guess the correct subdomains.
+If the provided server is any one of the following:
+
+```text
+http(s)://admin.dasch.swiss
+http(s)://app.dasch.swiss
+http(s)://api.dasch.swiss
+http(s)://iiif.dasch.swiss
+http(s)://dasch.swiss
+dasch.swiss
+```
+
+DSP-TOOLS will treat it as `https://api.dasch.swiss`,
+and derive the SIPI server URL `https://iiif.dasch.swiss` from it.
+
+This guessing feature comes with a price, though:
+
+- Only servers ending with `dasch.swiss` are supported.
+- If a server's configuration differs from the convention described above, DSP-TOOLS will fail.
```

Comment:
I think the last line is important. Perhaps not for now, but as soon as the infra team decides to silently modify this convention, DSP-TOOLS will break. So we should somewhere keep this info. 

Ideally, it would live in the Architectural Decision Records, but then I would have to create a subpage there, just for this info. If I leave it here, all infos belonging to this subject live at the same place. What do you think?